packages/objects/src/errors/InvalidQueryStatusError.ts

```diff
@@ -0,0 +1,9 @@
+import { BaseError } from "@objects/errors/BaseError.js";
+import errorCodes from "@objects/errors/errorCodes.js";
+
+export class InvalidQueryStatusError extends BaseError {
```

Comment:
Is this error a little too specific? I don't remember where I saw it used, but my blink is that either A. another error would be appropriate or B. "InvalidStatusError" might be better as we could use it other places.